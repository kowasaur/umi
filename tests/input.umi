il void print(string) "call void [mscorlib]System.Console::Write(string)"
il string input() "call string class [mscorlib]System.Console::ReadLine()"
il string +(string, string) "call string string::Concat(string, string)"

void main() {
    print("Enter name: ")
    print(("Hello " + input()) + "!\n")
}
