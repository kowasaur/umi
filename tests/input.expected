Enter name: Hello 0!
